pkg/tuple/v1_test.go

```diff
@@ -44,3 +46,26 @@ func TestJoinSubjectRef(t *testing.T) {
 		require.Equal(t, tt.expected, V1StringSubjectRef(tt.ref))
 	}
 }
+
+func TestBackAndForth(t *testing.T) {
+	for _, tc := range testCases {
+		tc := tc
+		if tc.stableCanonicalization == "" {
+			continue
+		}
+
+		t.Run("relationship/"+tc.input, func(t *testing.T) {
+			v1rel := ToV1Relationship(tc.relFormat)
+			testutil.RequireProtoEqual(t, tc.v1Format, v1rel, "mismatch in v1 proto")
+			adjusted := tc.relFormat
+
+			// Converting to V1 removes the timezones.
```

Comment:
Does this mean that when the user writes with timezone, and reads back via `ReadRelationships`, they get UTC timestamps? I think that's fine, but could make the life of the user harder if they are not properly handling timezones. Please document in PR body.

Reply:
Yes, especially because V1 does so

Reply:
what is V1?